frontend/src/pages/FormBCSCPage.vue

```diff
@@ -363,6 +366,22 @@ watch([validationError], () => {
     })
   }  
 });
+
+const districtsList = ref([]);
+useFetchTo("/api/clients/districts?page=0&size=250", districtsList);
```

Comment:
We changed the URL so it's now `/api/districts` instead of `/api/clients/districts`